src/main/java/pizzeria/data/Ingredient.java

```diff
@@ -1,37 +1,61 @@
 package pizzeria.data;
 
 /**
- * Enumerator for ingredients.
+ * This class is used to represent the ingredients per sucursal.

```

Comment:
"Represent the ingredients per store" directo y mantengamos un solo label "Store" no sucursal

Reply:
done at 2064659